converters/google/convert.go

```diff
@@ -291,10 +295,10 @@ func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
 	return nil
 }
 
-// For create/update, we need to handle both the case of no merging,
+// For create/update/no-op, we need to handle both the case of no merging,
 // and the case of merging. If merging, we expect both fetch and mergeCreateUpdate
 // to be present.
-func (c *Converter) addCreateOrUpdate(rc *tfjson.ResourceChange) error {
+func (c *Converter) addResourceChange(rc *tfjson.ResourceChange) error {
```

Comment:
Now that I'm seeing the implementation, I think maybe this should be `addCreateOrUpdateOrNoop` after all; it's longer but perhaps more clear (and matches `createOrUpdateOrNoops` above). But I think it could be fine either way (and it would be easy to change later since it's private). What do you think?

Reply:
`addCreateOrUpdateOrNoop` is definitely more clear. Renamed.